src/app/modules/main/module.nim

```diff
@@ -1570,4 +1570,7 @@ method insertMockedKeycardAction*[T](self: Module[T], cardIndex: int) =
 method removeMockedKeycardAction*[T](self: Module[T]) =
   self.keycardService.removeMockedKeycardAction()
 
+method fakeLoadingScreenFinished*[T](self: Module[T]) =
+  self.events.emit(SIGNAL_APP_FULLY_LOADED, Args())
```

Comment:
Maybe name the signal `FAKE_LOADING_SCREEN_FINISHED` instead? Because this is what this signal actually means 🤔